frontend/app/components/wp-resizer/wp-resizer.directive.ts

```diff
@@ -34,12 +34,15 @@ export class WorkPackageResizerController {
   private mouseMoveHandler:any;
 
   constructor(public $element:ng.IAugmentedJQuery) {
-    // Get element & starting width
+    // Get element
     this.detailsSide = <HTMLElement>document.getElementsByClassName('work-packages-split-view--details-side')[0];
-    this.elementFlex = localStorage.getItem("detailsSideFlexBasis") ? parseInt(localStorage.getItem("detailsSideFlexBasis")) : 582;
 
-    // Apply width if stored in local storage
+    // Get inital width from local storage and apply
+    let localStorageValue = localStorage.getItem("detailsSideFlexBasis");
+    this.elementFlex = localStorageValue ? parseInt(localStorageValue, 10) : 582;
     this.detailsSide.style.flexBasis = this.elementFlex + 'px';
+    // Apply two column layout
+    this.elementFlex > 700 ? this.detailsSide.classList.add('-columns-2') : this.detailsSide.classList.remove('-columns-2');
```

Comment:
You can use `classList.toggle('-columns-2', this.elementFlex > 700)` or `jQuery(this.detailsSide).toggleClass(...)` instead of the hard to read ternary operator